docs/basics/part-7-using-remote-runners.md

```diff
@@ -0,0 +1,88 @@
+Earthly has the ability to run builds both locally and remotely. If you followed the standard [installation instructions](https://earthly.dev/get-earthly), then you most likely have only run local builds so far. In this section, we will explore how to use remote runners to perform builds on remote machines.
+
+## Remote runners
+
+Earthly is able to use remote runners for performing builds on remote machines. When Earthly uses a remote runner, the inputs of the build are picked up from the local environment, then the execution takes place remotely, including any pushes (`RUN --push` commands, and `SAVE IMAGE --push` commands), but any local outputs are sent back to the local environment. All this takes place while your local Earthly process still provides the logs of the build in real time locally.
+
+Remote runners are especially useful in a few specific circumstances:
```

Comment:
nit: Something that I think Adam mentioned - you can build without needing to install docker.
It's a nice side benefit, perhaps it's something worth mentioning here.

Reply:
Adding to the remote runners page.